java/arcs/core/data/RawEntity.kt

```diff
@@ -73,36 +52,20 @@ data class RawEntity(
     /** Constructor for a [RawEntity] when only the field names are known. */
     constructor(
         id: ReferenceId = NO_REFERENCE_ID,
-        singletonFields: Set<FieldName> = emptySet(),
+        singletonFields: Set<FieldName>,
```

Comment:
did you mean to remove the default argument value here?

Reply:
if I provide a default for all arguments, the compiler complains that when I just do RawEntity() it cannot choose among the two constructors. So I have to leave at least one of these two without a default.